pkg/action/action.go

```diff
@@ -224,6 +224,19 @@ func (c *Configuration) renderResources(ch *chart.Chart, values chartutil.Values
 	return hs, b, notes, nil
 }
 
+func filterManifests(manifests []releaseutil.Manifest, skipTests bool) []releaseutil.Manifest {
+	if skipTests {
+		var manifestsWithoutTests []releaseutil.Manifest
+		for _, m := range manifests {
+			if !strings.Contains(m.Name, "tests/") {
```

Comment:
I would suggest looking for test hooks rather than searching within a test directory. Here's one way you could do this:

https://github.com/helm/helm/blob/c2da4fd53dc57189a46797a87b6b30dcd8d44879/pkg/action/release_testing.go#L81-L83

Reply:
Thanks for the pointer. I will look into it.